ocaml/idl/datamodel.ml

```diff
@@ -2187,7 +2187,13 @@ end
 module Tunnel = struct
   let tunnel_protocol =
     Enum
-      ("tunnel_protocol", [("gre", "GRE protocol"); ("vxlan", "VxLAN Protocol")])
+      ( "tunnel_protocol"
+      , [
+          ("gre", "GRE protocol")
+        ; ("vxlan", "VxLAN Protocol")
+        ; ("vxlan_mesh", "VxLAN Protocol for cross-server private network")
```

Comment:
A detailed description for the "mesh" would be better, like "setup vxlan tunnels to every other host in the pool"